ruby_ast_gen.gemspec

```
@@ -23,8 +23,9 @@ Gem::Specification.new do |spec|
   # Specify which files should be added to the gem when it is released.
   # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
   gemspec = File.basename(__FILE__)
-  spec.files = Dir.glob("{lib,exe,sig,vendor}/**/*") + %w[README.md LICENSE.txt Rakefile]
+  spec.files = Dir["lib/**/*.rb", "exe/**/*", "sig/**/*", "vender/**/*", "README.md", "LICENSE.txt", "RAKEFILE"]
```

```suggestion
  spec.files = Dir["lib/**/*.rb", "exe/**/*", "sig/**/*", "vendor/**/*", "README.md", "LICENSE.txt", "RAKEFILE"]
```
Also, commented out line at the bottom, doesn't that just stay? What is the difference besides only enforcing ruby files in `lib`?